solver-benchmarks/HackageBenchmark.hs

```diff
@@ -164,7 +164,7 @@ hackageBenchmarkMain = do
           if null argPackages
           then do
             putStrLn $ "Obtaining the package list (using " ++ argCabal1 ++ ") ..."
-            list <- readProcess argCabal1 ["list", "--simple-output"] ""
+            list <- readProcess argCabal1 ["list", "--simple-output", "--exact"] ""
```

Comment:
This doesn't seem to be related to the reported issue. Why this change? Does `exact` makes it somehow faster?